tests/datasets/commands_tests.py

```diff
@@ -343,3 +345,32 @@ def test_import_v1_dataset_validation(self):
                 "database_name": ["Missing data for required field."],
             }
         }
+
+    def test_import_v1_dataset_existing_database(self):
```

Comment:
This unit test is unrelated, but I thought about this case and decided to add it in this PR.